src/libraries/System.ComponentModel.TypeConverter/tests/ILLink.Descriptor.xml

```diff
@@ -0,0 +1,21 @@
+<!-- This is file is to tell the linker not to remove these assemblies/types/namespace 
```

Comment:
The name of the file should follow the linker naming pattern. There should be an `s` in `Descriptors`.

`ILLink.Descriptors.xml`

See the docs here:
https://github.com/mono/linker/blob/55bd0ebd00ddbf2cd817314b1cf6824003cfb463/docs/data-formats.md#descriptor-format